src/style/mixins.scss

```diff
@@ -130,6 +130,30 @@
   padding: 0;
 }
 
+/// Visually hides an element but still allows it to be tabbed to.
+@mixin iui-visually-hidden {
+  clip-path: inset(50%);
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  height: 1px;
+  width: 1px;
+  padding: 0;
+  margin: -1px;
+  border-width: 0;
+}
+
+@mixin iui-visually-hidden-revert {
```

Comment:
Any reason you added the `iui-` prefix? I would think this should be an internal mixin.

Also maybe add a comment above this mixin explaining what it does and that it is recommended to use on wrappers.

Reply:
Removed `iui-` prefix & added comments.